packages/fuels-contract/src/script.rs

```diff
@@ -22,7 +35,213 @@ impl Script {
         Self { tx }
     }
 
-    // Calling the contract executes the transaction, and is thus state-modifying
+    pub async fn from_call(call: &ContractCall) -> Self {
```

Comment:
docstring (even if the name is self-documenting, a summary of what the whole function does is in order here)